Build failed following Spin quickstart document

OS: Windows 11 Professional
cargo --version
cargo 1.62.0 (a748cf5a3 2022-06-08)

Steps to reproduce:
$ git clone https://github.com/fermyon/spin -b v0.4.0
$ cd spin
$ rustup target add wasm32-wasi
$ cargo install --path .

Error message:
`error[E0308]: mismatched types
   --> src\commands\deploy.rs:181:13
    |
181 |             ChannelRevisionSelectionStrategy::UseRangeRule,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `hippo_openapi::models::channel_revision_selection_strategy::ChannelRevisionSelectionStrategy`, found enum `ChannelRevisionSelectionStrategy`
    |
    = note: perhaps two different versions of crate `hippo_openapi` are being used?

error[E0609]: no field `apps` on type `hippo_openapi::models::app_item_page::AppItemPage`
   --> src\commands\deploy.rs:255:27
    |
255 |         let app = apps_vm.apps.iter().find(|&x| x.name == name.clone());
    |                           ^^^^ unknown field
    |
    = note: available fields are: `items`, `total_items`, `page_index`, `page_size`, `is_last_page`

Some errors have detailed explanations: E0308, E0609.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `spin-cli` due to 2 previous errors
error: failed to compile `spin-cli v0.4.0 (C:\dev\spin)`, intermediate artifacts can be found at `C:\dev\spin\target``

--- a/docs/content/quickstart.md
+++ b/docs/content/quickstart.md
@@ -27,7 +27,7 @@ If you have [`cargo`](https://doc.rust-lang.org/cargo/getting-started/installati
 $ git clone https://github.com/fermyon/spin -b v0.4.0
 $ cd spin
 $ rustup target add wasm32-wasi
-$ cargo install --path .
+$ cargo install --locked --path .
 $ spin --help
 ```
 

